### ISSUE TITLE
Game rarely starts and runs awfully

### ISSUE THREAD

**Issue by ollietheotter983** [o5k/tetrio-issues#1414]
Puyo Puyo Tetris 2 just about runs fine on here but tetrio is a nightmare. yes ik its bad hardware but cmon. its the windows 8.1 desktop client.

![20241107_221947.jpg](https://github.com/user-attachments/assets/68974410-27a6-4187-bc07-bfa89af189bb)

**Comment by ZaptorZap** [comment]
We would likely need to see what's displayed in the `Display` tab to gauge graphics performance, although there may not even be a GPU to inspect. That said, a 2 core laptop APU from early 2013 is pretty desolate in the year 2025. I know "it's just a block game", and we're truly looking at improving performance to reflect that; but there's only so much we can support right now.

**Comment by ollietheotter983** [comment]
Yeah I do understand such awful hardware isn't a priority lol, just trying to make the most of what I have
